### PR TITLE
Adding Haploclique, version 1.3.1

### PR DESCRIPTION
**Pull request by DrYak**
Adding the latest piece of the jigsaw puzzle required by V-pipe.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by DrYak** [comment]
Fulfilling the request from [this issue](https://github.com/cbg-ethz/haploclique/issues/35)

**Comment by DrYak** [comment]
Travis tests passed, yay !

File: recipes/haploclique/1.3.1/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{% set name = "haploclique" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "2ea8c6089d3f971c0fd5d20e708905521e1b6f802a8835d2cd2817f48e442a9c" %}
+
+about:
+    summary: Viral quasispecies assembly via maximal clique finding. A method to reconstruct viral haplotypes and detect large insertions and deletions from NGS data.
+    home: https://github.com/cbg-ethz/{{ name }}
+    license: GPLv3
```
Comment (bgruening) [inline recipes/haploclique/1.3.1/meta.yaml]: Can we add lincese_file: as well please. GPL want to have a copy of the license distributed with the software.

Comment (DrYak) [reply recipes/haploclique/1.3.1/meta.yaml]: Okay, I'll ask @armintoepfer upstream.

Comment (armintoepfer) [reply recipes/haploclique/1.3.1/meta.yaml]: @DrYak please refer to your PI about the license.

Comment (DrYak) [reply recipes/haploclique/1.3.1/meta.yaml]: Isn't the code already licensed under "[*GNU General Public License* {...} *version 3* {...} *or* {...} *any later version*](https://github.com/cbg-ethz/haploclique/blob/7707f1ab143f2c2b4b7b3b01bc51f86d53ac710e/src/haploclique.cpp#L6)" ?

Or shall I re-check with the Professor ?

Comment (armintoepfer) [reply recipes/haploclique/1.3.1/meta.yaml]: In that case, please make a PR for the license

Comment (DrYak) [reply recipes/haploclique/1.3.1/meta.yaml]: I've [just done it](https://github.com/cbg-ethz/haploclique/pull/61)

Comment (armintoepfer) [reply recipes/haploclique/1.3.1/meta.yaml]: Merged

**Comment by DrYak** [comment]
Here's the [upgraded](https://github.com/DrYak/bioconda-recipes/blob/82f88b28376cc6595293a88f1c9c35454b79349e/recipes/haploclique/1.3.1/meta.yaml#L10) pull-request

**Comment by DrYak** [comment]
Travis tests passing with the new pull-request, yay !

**Comment by DrYak** [comment]
@bgruening , you think everything is okay, or do you still think that some other changes are needed ?

**Comment by DrYak** [comment]
Who would have the write access to merge this PR into bioconda ?
@armintoepfer ? @SoapZA  ?

**Comment by armintoepfer** [comment]
If there are no complaints until tmrw morning, I'll merge, if not yet done.

**Comment by DrYak** [comment]
(does the happy github pull-merge dance)

**Comment by armintoepfer** [comment]
https://travis-ci.org/bioconda/bioconda-recipes/jobs/308895016

**Comment by mbargull** [comment]
#6950

**Comment by armintoepfer** [comment]
Ok! Good to know that we didn't break it. Thank you

**Comment by DrYak** [comment]
(It would have been weird if a stand-alone recipe did pass travis while in the pull-request, but broke everything while merged).

I hope that this will get fixed soon.

(At least, I'm done with adding cbg's software to bioconda recipes for now, so I won't be affected personally by this)

**Pull request merged**